README.md

```diff
@@ -59,7 +59,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+<!-- ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [ ] It's a requirement that we should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." as it is placeholder text. This applies to line `103` as well n the ReadMe file.